.golangci.yml

```diff
@@ -3,6 +3,16 @@ run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
   timeout: 1m
 
+issues:
+  # Maximum count of issues with the same text.
+  # Set to 0 to disable.
+  # Default: 3
+  max-same-issues: 0
+  # Maximum issues count per one linter.
+  # Set to 0 to disable.
+  # Default: 50
+  max-issues-per-linter: 0
```

Comment:
I personally prefer to see the entire list of issues at once and go through them without re-running the linter all the time.